warehouse/warehouse.go

```diff
@@ -1106,6 +1341,8 @@ func startWebHandler() {
 	if isMaster() {
 		backendconfig.WaitForConfig()
 		http.HandleFunc("/v1/process", processHandler)
+		http.HandleFunc("/v1/pending-events", pendingEventsHandler)
```

Comment:
let's make it `/v1/warehouse/pending-events`. Should have done it ideally for /process too but we should start doing it for new routes